pyproject.toml

```
@@ -177,7 +177,7 @@ package = { "cwd" = "_packaging", shell = "pyinstaller -y gaphor.spec"}
 win-installer = { "script" = "_packaging.windows.build-win-installer:main" }
 appimage-docker-build = "docker build -t ghcr.io/gaphor/gaphor-appimage _packaging/appimage"
 appimage-docker-build-clean = "docker build --no-cache -t ghcr.io/gaphor/gaphor-appimage _packaging/appimage"
```

```suggestion
appimage-docker-build-clean = "docker build --no-cache -t ghcr.io/gaphor/gaphor-appimage:2023.1.0 _packaging/appimage"
```

The built docker image needs the same tag as the one we push.